sanctions/apps/sanctions/tests/test_models.py

```diff
@@ -0,0 +1,257 @@
+"""
+Tests for Sanctions models.
+"""
+from datetime import datetime
+
+from django.test import TestCase
+from testfixtures import LogCapture
+
+from sanctions.apps.sanctions.models import SanctionsCheckFailure, SDNFallbackData, SDNFallbackMetadata
+from sanctions.apps.sanctions.tests.factories import SDNFallbackDataFactory, SDNFallbackMetadataFactory
+
+
+class SanctionsCheckFailureTests(TestCase):
+    """
+    Tests for SanctionsCheckFailure model.
+    """
+    def setUp(self):
+        super().setUp()
+        self.full_name = 'Keyser Söze'
+        self.username = 'UnusualSuspect'
+        self.country = 'US'
+        self.sdn_check_response = {'description': 'Looks a bit suspicious.'}
+
+    def test_unicode(self):
+        """ Verify the __unicode__ method returns the correct value. """
+        hit = SanctionsCheckFailure.objects.create(
+            full_name=self.full_name,
+            username=self.username,
+            country=self.country,
+            sanctions_type='SDN',
```

Comment:
in my PR is used `ISN,SDN` for this value, because i think that's the list we hand the API. not sure if this is worth changing or not

Reply:
It will matter when we need to query this data, but not on the test IMO, but yes let's make sure the same value is used from the client. Which PR are you talking about? Merged?

This is a new field not present in ecommerce, not to be confused with `sdn_type` for the SDNFallbackData, which is `'individual'`.

Reply:
Updated the example on `models.py` as well.

Reply:
this is the pr: https://github.com/edx/sanctions/pull/14

noted about the different of `sdn_type` for `SDNFallbackData` 👍

Reply:
Ah I see it where you got that from, from the `sdn_api_list` in the SDNClient right?

Reply:
yep